### ISSUE TITLE
Possible bug: changing the default value of slot named `hints-alphabet` via "Configure" button (GUI) does not work

### ISSUE THREAD

**Issue by pdelfino** [atlas-engineer/nyxt#1774]
**Describe the bug**
Using the Nyxt's GUI to change the default value of `hints-alphabet` does change the `auto-config.lisp` file. However, it does not generate the expected behavior. Moreover, it blocks Nyxt from starting.

**Precise recipe to reproduce the issue**
- start Nyxt
- execute command `describe-slot`;
- type `link-alphabet`;
- select the second option, from `user-browser` class;
- click on the `Configure` button;
- insert the string (with quotes): `"asfdghjkl"`
- see that the local file `auto-config.lisp` has changed with the following expression included:
```lisp
(DEFINE-CONFIGURATION NYXT/WEB-MODE:USER-WEB-MODE
  ((NYXT/WEB-MODE:HINTS-ALPHABET "asfdghjkl")))
```
- look at the REPL, it shows this message:
```lisp
end of file on #<SB-IMPL::STRING-INPUT-STREAM {100A66A473}>
   [Condition of type END-OF-FILE]
```
- close Nyxt
- quit Slime/Sly;
- start Slime
- load the system nyxt/gi-gtk;
- try starting Nyxt
-  an error occurs with the following message:
```lisp
define-configuration argument USER-USER-WEB-MODE is not a known class.
   [Condition of type SIMPLE-ERROR]
```

Observation: if I remove the new addition on `auto-config.lisp` restoring how it used to be and if I simply insert the same expression in my **init.lisp** it **does** work:

```lisp
(DEFINE-CONFIGURATION NYXT/WEB-MODE:USER-WEB-MODE
  ((NYXT/WEB-MODE:HINTS-ALPHABET "asfdghjkl")))
  ```

For website-specific issues:
Can you reproduce this issue with Epiphany / GNOME Web (https://wiki.gnome.org/Apps/Web)?
Not relevant.

**Information**
- OS name+version: 
Ubuntu 20.04 LTS

- Graphics card and driver:
product: Iris Plus Graphics 640
configuration: driver=i915 latency=0

- Desktop environment / Window manager name+version:
Name: GNOME Shell 3.36.7
Window manager's "showing the desktop" mode: OFF

- How you installed Nyxt (Guix pack, package manager, build from source):
Source

- Information from command `show-system-information`:

Since Nyxt is not starting, I cannot do it.

  If you can't run `show-system-information`, try `nyxt --system-information` from
  a shell.  It this still does not work, please provide the following:
 
  - Nyxt version (from the `nyxt-version` command or `nyxt --version` from a shell): 
  
 ```lisp
  NYXT> (nyxt-version)
 <INFO> [18:27:54] nyxt message.lisp (echo form-fun-6) -
  Version 2.1.1-607-g0bafa5cc
NIL 
 ```
  
 - Lisp implementation/version (if built from source):  This is SBCL 2.0.1.debian

  - Kernel name+version:  
  ```bash
$ cat /proc/version
Linux version 5.11.0-27-generic (buildd@lcy01-amd64-019) (gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0, GNU ld (GNU Binutils for Ubuntu) 2.34) #29~20.04.1-Ubuntu SMP Wed Aug 11 15:58:17 UTC 2021
```

  - WebKitGTK+ | QtWebEngine version: 
  ```bash
  $ dpkg -s libgtk-3-0|grep '^Version'
Version: 3.24.20-0ubuntu1
```
**Output when started from a shell**
It was run from source in the Slime's REPL.

**Comment by floyza** [comment]
Looks like the same problem as #1156

**Comment by Ambrevar** [comment]
Should be fixed with #2194.

**Comment by Ambrevar** [comment]
@pdelfino Please reopen if this is still an issue.
